app/lib/ui/views/home_screen.dart

```diff
@@ -41,12 +48,29 @@ class HomeScreen extends StatelessWidget {
                           onPressed: model.navigateToAccountScreen,
                         )),
                     Positioned(
-                        child: CircleAvatar(
-                      radius: 70,
-                      backgroundColor: Colors.white,
-                      backgroundImage: NetworkImage(
-                          "https://images.unsplash.com/photo-1544393569-eb1568319eef?ixid=MXwxMjA3fDB8MHxzZWFyY2h8MXx8c3RvcHxlbnwwfHwwfA%3D%3D&ixlib=rb-1.2.1&w=1000&q=80"),
-                    ))
+                      child: model.transporter != null
+                          ? CircleAvatar(
+                              radius: 52,
+                              backgroundColor: NavyBlue,
+                              child: ClipRRect(
+                                borderRadius: BorderRadius.circular(50),
+                                child: Container(
+                                  color: Colors.grey[300],
+                                  child: Image.network(
+                                    model.transporter.displayImageUrl,
```

Comment:
- [ ] This will fail maybe if the image URL is null, put something else here when the transporter does not have a URL

Reply:
Wait a moment, is `Image.network(null)` valid? Maybe this already works fine.

Reply:
Nope. It must not be null:
```
  /// Creates an object that fetches the image at the given URL.
  ///
  /// The arguments [url] and [scale] must not be null.
```

Reply:
If transporter have a null displayImageUrl, then that means user did not upload his/her picture (which is fine).

To solve this issue, I added a default image inside assets and will be used if displayImageUrl is empty.